src/test/scala/com/codacy/client/bitbucket/TestUtils.scala

```diff
@@ -0,0 +1,19 @@
+package com.codacy.client.bitbucket
+
+import scala.io.Source
+
+object TestUtils {
+
+  /**
+    * Helper function which will read the content of a file into a string. Used to test JSON responses.
+    * @param path the relative path of the json file to consume
+    */
+  def getTestContent(path: String): String = {
+    val testFile = Source.fromFile(path)
```

Comment:
Can we try "val testFile = Source.fromURL(getClass.getResource(path))"

And with this, you should be able to call with getTestContent("/user-service/user_emails.json")

Reply:
Maybe @andreaTP has any insight to add here?

Reply:
Ah nice, I was looking for a better way of calling that

Reply:
should be something like:
```scala
Source.fromFile(getClass.getResource(path).getFile())
```

Reply:
Do you prefer to get a .getFile to be able to use the fromFile? We can also use the fromURL then? I am fine with either